agent/fw_drupal8.c

```diff
@@ -685,6 +685,8 @@ void nr_drupal_version() {
   if (nr_php_is_zval_valid_string(zval_version)) {
     char* version = Z_STRVAL_P(zval_version);
     nr_txn_add_php_package(NRPRG(txn), "drupal/core", version);
+    nr_fw_support_add_package_supportability_metric(NRPRG(txn), "drupal/core",
```

Comment:
Perhaps "drupal/core" would be a good candidate for a C macro to remove duplication of this constant and so the compiler would catch if the macro was spelled wrong and flag the error.  Whereas if the constant string here was ever modified incorrectly the compiler would still work but we'd send the wrong string.

Applies for all these string constants for the package "name" in this PR.

Reply:
Good idea, added: [1fb5879](https://github.com/newrelic/newrelic-php-agent/pull/868/commits/1fb5879e914e31768f86c12a0ce99907a70f65a2)